internal/pkg/manifest/validate.go

```diff
@@ -151,6 +152,18 @@ func (l LoadBalancedWebService) validate() error {
 	}); err != nil {
 		return fmt.Errorf("validate unique exposed ports: %w", err)
 	}
+	exposedPortsIndex, err := l.ExposedPorts()
+	if err != nil {
+		return err
+	}
+	if err = validateHealthCheckPorts(validateHealthCheckPortsOpts{
+		exposedPorts:      exposedPortsIndex,
+		mainContainerPort: l.ImageConfig.Port,
+		alb:               &l.HTTPOrBool.HTTP,
+		nlb:               &l.NLBConfig,
+	}); err != nil {
+		return fmt.Errorf("validate load balancer health check ports: %w", err)
```

Comment:
Does it make more sense to be
```suggestion
	if err = validateALBHealthCheckPorts(validateHealthCheckPortsOpts{
		exposedPorts:      exposedPortsIndex,
		mainContainerPort: l.ImageConfig.Port,
		alb:               &l.HTTPOrBool.HTTP,
	}); err != nil {
		return fmt.Errorf("validate application load balancer health check ports: %w", err)
```
And do the same thing to NLB separately? Because `validateHealthCheckPorts` seems to deal ALB and NLB separately.

Reply:
I don't think this is a good idea because it could lead to the error messages being slightly misleading. When a health check is exposed on a port using udp, the solution is _either_ to change the NLB port exposing udp, or to change the health check port set to the port using UDP traffic. Because the solution can be approached from either direction I think it's more logically sound to validate them together.